Tests/TestingApps/PaywallsTester/PaywallsTester/UI/Views/SamplePaywallsList.swift

```diff
@@ -95,7 +95,7 @@ struct SamplePaywallsList: View {
                 paywallComponentsData: data,
                 offering: Self.loader.offeringWithDefaultPaywall(),
                 onDismiss: { self.display = nil }
-            )
+            ).environmentObject(Self.purchaseHandler)
```

Comment:
I don't think this is how we wanna do it (permanently)? It's a separate commit, so I can easily revert it.

Reply:
Ah yeah, I fixed this is one of my packages commits! So if you want to remove this commit so the merge is clean that would be 🔥 

But thanks for calling this out 🫶

Reply:
Totally! Revert is here: https://github.com/RevenueCat/purchases-ios/pull/4372/commits/985b78a4aaa8b65e876ed990fc711786818b8ddc.